cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/AzureCloudSubnetParametersService.java

```diff
@@ -0,0 +1,27 @@
+package com.sequenceiq.cloudbreak.cloud.azure;
+
+import java.util.Optional;
+
+import org.springframework.stereotype.Service;
+
+import com.sequenceiq.cloudbreak.cloud.model.CloudSubnet;
+
+@Service
+public class AzureCloudSubnetParametersService {
+
+    public static final String PRIVATE_ENDPOINT_NETWORK_POLICIES = "privateEndpointNetworkPolicies";
+
+    private static final String ENABLED = "enabled";
+
+    private static final String DISABLED = "disabled";
+
+    public void addPrivateEndpointNetworkPolicies(CloudSubnet cloudSubnet, String state) {
+        cloudSubnet.putParameter(PRIVATE_ENDPOINT_NETWORK_POLICIES, "Disabled".equalsIgnoreCase(state) ? DISABLED : ENABLED);
```

Comment:
I wanted to decouple what Azure sends back and our internal representation, that's why I did not merge the two.

Reply:
I could add it as another constant to make it clear that this is the string we expect from Azure.

Reply:
I am saying that the internal representation can be the same as the Azure value, it has no disadvantage in this case

Reply:
Only thing that it is persisted in env service. If azure representation changes then it would be enough to decouple internal representation from the azure one but doing it now does not do any harm in my opinion, it helps future devs.